docs/api/channels.rst

```diff
@@ -182,6 +182,24 @@ ForumTag
     :members:
     :inherited-members:
 
+ThreadWithMessage
+~~~~~~~~~~~~~~~~~
+
+.. class:: ThreadWithMessage
+
+    A namedtuple which represents a thread and message returned from :meth:`ForumChannel.create_thread`.
```

Comment:
```suggestion
    A :class:`~typing.NamedTuple` which represents a thread and message returned from :meth:`ForumChannel.create_thread`.
```

Reply:
I originally took this phrasing from `BanEntry`, so I've changed the reference there as well: https://github.com/DisnakeDev/disnake/pull/1215/commits/9f8137232708469ed1207b5eaecfd4c3798d254f